python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -127,6 +143,10 @@ def _graph_data_to_fused_csc_sampling_graph(
         del coo_src_list
         coo_dst = torch.cat(coo_dst_list)
         del coo_dst_list
+        if len(edge_type_to_id) <= torch.iinfo(torch.int32).max:
```

Comment:
Why don't we use even smaller integer types here for the edge_types? We can use `torch.uint8`, `torch.int16`, `torch.int32` and `torch.int64`

Reply:
Our sampling code has support for these smaller integer types.